README.md

```diff
@@ -61,6 +62,63 @@ The options include:
 * `serverId` ID of the server to restrict the tests against.
 * `serversUrl` URL to obtain the list of servers available for speed test. (default: http://www.speedtest.net/speedtest-servers-static.php)
 
+## Proxy by env
+
+### You can set proxy with env var
+
+Code use example for cli:
+```bash
+$ npm install --global speedtest-net
+$ set HTTP_PROXY=http://proxy:3128
+$ speedtest-net
+```
+
+Code use example for node:
+```js
+process.env["HTTP_PROXY"] =  process.env["http_proxy"] = "http://proxy:3128";
+//process.env["HTTPS_PROXY"] = process.env["https_proxy"] = "https://proxy:3128";
+
+const speedTest = require('speedtest-net');
```

Comment:
Since `speedtest` is quite old and uses `var` everywhere, I think it would be best not to use `const` in the docs. Also, using `var` makes this module compatible with older node versions (which may still run in many production environments).

Besides that, I'm all for `const` and `let`.

Reply:
hello i respect your example : https://github.com/ddsol/speedtest.net

:)

Reply:
Yeah, my bad. If you want to, you can fix that one as well 😄.

Reply:
done

Reply:
done